companion-files.md

```diff
@@ -0,0 +1,32 @@
+## Companion files
+
+Discussed Spring 2016, will require new versions of notebook and ?nbformat.
```

Comment:
That's deliberate, if a bit ugly - I'm not quite sure if it will require a new nbformat.

It depends whether we consider the companion files part of the specified notebook file format, or as an additional thing that the notebook server loads separately.